datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -336,9 +336,10 @@ def metadata(check, check_duplicates, show_warnings):
 
             # metric_name header
             if metric_prefix:
-                if not row['metric_name'].startswith(metric_prefix):
-                    prefix = row['metric_name'].split('.')[0]
-                    metric_prefix_count[prefix] += 1
+                prefix = row['metric_name'].split('.')[0]
+                if prefix not in ['system', 'jvm', 'http', 'datadog', 'sftp']:
```

Comment:
probably doesn't matter but could use a `set` to perform this lookup.

Reply:
I can move this to a constant